lua/kubectl/views/fallback/definition.lua

```diff
@@ -109,6 +110,8 @@ function M.getHeaders(rows)
     local firstRow = rows.rows[1]
     if firstRow and firstRow.object and firstRow.object.metadata and firstRow.object.metadata.namespace then
       table.insert(headers, "NAMESPACE")
+    else
+      M.namespaced = false
```

Comment:
You think we can use this information instead?

We already know if a resource is namespaced or not by your other code in https://github.com/Ramilito/kubectl.nvim/blob/41333f68718e6ae4e26d67716cd091f01a854b53/lua/kubectl/views/definition.lua#L109

So if we add that into the cached_api_resource object, maybe we can simplify this code a bit

Reply:
you are right.
I did what you asked, though I'm not sure it's enough, can you please check again and let me know if there's anything you'd change? @Ramilito

Reply:
Il take a look soon 👍